StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/GenericInfoScreen/GenericInfoBodyView.swift

```diff
@@ -0,0 +1,163 @@
+//
+//  GenericInfoBodyView.swift
+//  StripeFinancialConnections
+//
+//  Created by Krisjanis Gaidis on 7/11/24.
+//
+
+import Foundation
+import UIKit
+
+func GenericInfoBodyView(
+    body: FinancialConnectionsGenericInfoScreen.Body?,
+    didSelectURL: @escaping (URL) -> Void
+) -> UIView? {
+    guard  let body, !body.entries.isEmpty else {
+        return nil
+    }
+    let verticalStackView = UIStackView()
```

Comment:
Should this be a `HitTestStackView`, since the text component might have URLs?

Reply:
great catch! fixed: https://github.com/stripe/stripe-ios/pull/3767/commits/52ad9599e14f820c2e571c8ab1b1b51f368d8938